MaintenanceManager/MaintenanceManager.cpp

```diff
@@ -585,23 +578,17 @@ namespace WPEFramework {
                             m_notify_status=MAINTENANCE_ERROR;
```

Comment:
Pls change m_notify_status to notify_status to indicate it's a local variable. Also pls change g_notify_status to m_notify_status to indicate it's a member and not global.

Reply:
fixed